Coordinator/Coordinator/Coordinator/AuthCoordinator.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  LoginCoordinator.swift
```

Comment:
```suggestion
//  LoginCoordinator.swift
```
название файла должно совпадать с классом